docs/install/query.md

```diff
@@ -0,0 +1,41 @@
+# Run your first query using RubiX
+
+## Start Hive Client
+
+    hive --hiveconf hive.metastore.uris="" --hiveconf fs.rubix.impl=com.qubole.rubix.hadoop2.CachingNativeS3FileSystem
+
+## Create External Table
+
+    CREATE EXTERNAL TABLE wikistats_orc_rubix 
+    (language STRING, page_title STRING,
+    hits BIGINT, retrived_size BIGINT)
+    STORED AS ORC
+    LOCATION 'rubix://emr.presto.airpal/wikistats/orc';
+
+## Run Query (Presto or Hive CLI)
+
+    SELECT language, page_title, AVG(hits) AS avg_hits
+    FROM default.wikistats_orc_rubix
+    WHERE language = 'en'
+    AND page_title NOT IN ('Main_Page',  '404_error/')
+    AND page_title NOT LIKE '%index%'
+    AND page_title NOT LIKE '%Search%'
+    GROUP BY language, page_title
+    ORDER BY avg_hits DESC
+    LIMIT 10;
+
+## Rubix Stats (supported on Presto only)
```

Comment:
do you want to keep consistence "Rubi**X**" ?

Reply:
Done